src-php/Fields/CloudinaryImage.php

```diff
@@ -29,6 +29,14 @@ public function __construct($name, $attribute = null, $disk = 'cloudinary', $sto
             return $this->value ? cloudinary_image($this->value, [
                 'width' => 318,
             ], $this->disk) : null;
+        })->download(function () {
+            $image_address = cloudinary_image($this->value, [
+                "width" => 1.0,
+                "height" => 1.0,
+            ]);
+            return response()->streamDownload(function () use ($image_address) {
+                echo file_get_contents($image_address);
+            }, 'image-download.jpg');
```

Comment:
Nice one @crumb1e ! Can we do something about this hard-coded filename and extension? Chances are not every file will be a jpg or even an image?

Reply:
Would this not work?

`
->download(function ($request, $model) {
            $image_address = cloudinary_image($this->value);

            return response()->streamDownload(function () use ($image_address) {
                echo file_get_contents($image_address);
            }, $this->value);
        })
`